src/locales/en-US/commands/reddit.json

```diff
@@ -2,29 +2,29 @@
 	"name": "reddit",
 	"description": "Get information from Reddit",
 	"subredditName": "subreddit",
-	"subredditDescription": "Get a random post from a specified subreddit.",
+	"subredditDescription": "Get a random post from a specified subreddit",
 	"postName": "post",
-	"postDescription": "Get the information of a specific post.",
+	"postDescription": "Get the information of a specific post",
 	"optionsRedditName": "reddit",
-	"optionsRedditDescription": "The subreddit name to get a random post from. The `/r/` and `r/` prefixes are also supported.",
+	"optionsRedditDescription": "The subreddit name to get a random post from.",
 	"optionsPostName": "post",
-	"optionsPostDescription": "A URL to a reddit post.",
-	"invalidName": "Subreddits must be between 3 and 21 characters long, and contain only letters, numbers, or underscores. The `/r/` and `r/` prefixes are also supported.",
-	"invalidPostKey": "Posts must have a key of at least 6 characters, and only container letters and numbers.",
-	"banned": "The specified subreddit is not deemed to be safe, please try another one.",
+	"optionsPostDescription": "A URL to a reddit post",
+	"invalidName": "Subreddits must be between 3 and 21 characters long, and contain only letters, numbers, or underscores. The `/r/` and `r/` prefixes are also supported",
```

Comment:
Only the elements in the slash UI must go without a final dot, not the responses. Please restore them.